components/build/tekton-chains/chains-certs.yaml

```diff
@@ -0,0 +1,82 @@
+apiVersion: v1
+kind: Secret
+metadata:
+  name: chains-ca-cert
+  namespace: tekton-chains
+type: Opaque
+---
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: chains-secrets-admin
+  namespace: tekton-chains
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: Role
+metadata:
+  name: chains-secret-admin
+  namespace: tekton-chains
+rules:
+- apiGroups:
+  - ""
+  resources:
+  - secrets
+  verbs:
+  - list
+  - create
+  - get
+  - update
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: RoleBinding
+metadata:
+  name: chains-secret-admin
+  namespace: tekton-chains
+roleRef:
+  apiGroup: rbac.authorization.k8s.io
+  kind: Role
+  name: chains-secret-admin
+subjects:
+- kind: ServiceAccount
+  name: chains-secrets-admin
+  namespace: tekton-chains
+---
+apiVersion: batch/v1
+kind: Job
+metadata:
+  name: chains-certs-configuration
+  namespace: tekton-chains
+  annotations:
+    argocd.argoproj.io/hook: Sync
+    argocd.argoproj.io/hook-delete-policy: HookSucceeded
+spec:
+  template:
+    spec:
+      containers:
+        - image: registry.redhat.io/openshift4/ose-cli:v4.9
+          command:
+            - /bin/bash
+            - -c
+            - |
+              echo "Waiting for secret and configmaps to be present"
+              until oc get secret chains-ca-cert -n tekton-chains &&\
+                oc get cm config-trusted-cabundle -n tekton-chains &&\
+                oc get cm kube-root-ca.crt -n tekton-chains
+              do
+                sleep 5;
+              done
+
+              echo "Update certificate"
```

Comment:
It would be great to have a more descriptive message, and maybe also a comment explaining why we need to generate a custom ca bundle for things to work.

Reply:
Updated.